console/module/device/src/main/java/org/eclipse/kapua/app/console/module/device/client/device/command/DeviceTabCommand.java

```diff
@@ -218,10 +219,10 @@ public void handleEvent(FormEvent be) {
                     if (UNDEFINED_ERROR.equals(errorMessage)) {
                         errorMessage = DEVICE_MSGS.deviceConnectionError();
                     } else if (errorMessage.contains(INTERNAL_ERROR)) {
-                        errorMessage = errorMessage.substring(errorMessage.indexOf(INTERNAL_ERROR) + INTERNAL_ERROR.length() + 1,
-                                errorMessage.indexOf(": error="));
+                        errorMessage = DEVICE_MSGS.deviceCommandExecutionErrorMessage();
                     }
-                    MessageBox.alert(MSGS.error(), MSGS.commandExecutionFailure() + ":<br/>" + errorMessage, null);
+                    StyleInjector.inject(".x-window-dlg .ext-mb-icon {width: 50px; height: 50px;}"); 
```

Comment:
Is this the only way?

Reply:
I think that we can change this to be a `InfoDialog`. Would make sense?